jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -93,6 +94,8 @@ class CodeWhispererService {
             }
             return
         }
+        val isInjectedFile = runReadAction { psiFile.isInjectedText() }
+        if (isInjectedFile) return
```

Comment:
do we want to show anything on manual trigger?

Reply:
this bug doesn't happen for manual trigger, not sure the reason but manual trigger always work

Reply:
interesting case, so manual trigger is able to display inlay where auto doesn't

Reply:
it must be that at the entry points the editor for auto trigger is a different one than the editor for manual trigger